documentation/docs/contribute/01-contribution-guide.md

```diff
@@ -1,4 +1,4 @@
-# Contributing
+# How to start contributing
```

Comment:
There is a mismatch between file name and header that could be confusing

Reply:
fixed